### PR TITLE
chore(weave): add tabs, and template ui to sidebar

### PR DESCRIPTION
**Pull request by jwlee64**
https://github.com/wandb/weave/assets/25037002/fa49dec6-5aa5-4362-a80a-f0c5d5c9ae7b

### CODE REVIEW

File: weave-js/src/components/PagePanelComponents/Home/HomePreviewSidebar.tsx [hunk 1]
```diff
@@ -148,21 +192,111 @@ export const HomePreviewSidebarTemplate: React.FC<{
   );
 };
 
+const Loader = () => (
```
Comment (jwlee64) [inline weave-js/src/components/PagePanelComponents/Home/HomePreviewSidebar.tsx:195]: This is pulled from lower in the file, just made it into a component so i could use it multiple spots

File: weave-js/src/components/PagePanelComponents/Home/HomePreviewSidebar.tsx [hunk 2]
```diff
@@ -210,101 +344,221 @@ export const HomeExpressionPreviewParts: React.FC<{
...
-                      refinedExpression.result as any,
-                      newDashExpr => {
-                        navigateToExpression(newDashExpr);
-                        setIsGenerating(false);
-                      }
-                    );
-                  }}
-                />
-              ))}
+              {recommendedTemplateInfo.op_name !== SEED_BOARD_OP_NAME &&
```
Comment (jwlee64) [inline weave-js/src/components/PagePanelComponents/Home/HomePreviewSidebar.tsx:366]: replaces the list of all templates with a recommended template, and the seed board template

File: weave-js/src/components/PagePanelComponents/Home/HomePreviewSidebar.tsx [hunk 3]
```diff
@@ -210,101 +344,221 @@ export const HomeExpressionPreviewParts: React.FC<{
...
+        style={{
+          gap: '8px',
+          paddingBottom: '32px'
+        }}>
+        <Label style={{
+          marginBottom: '8px',
+        }}>
+          Weave analyzes your data schema to suggest relevant board templates. Select a template to instantly generate a board.
+        </Label>
+        {recommendedTemplateInfo && (
```
Comment (jwlee64) [inline weave-js/src/components/PagePanelComponents/Home/HomePreviewSidebar.tsx:449]: shows the recommended template expanded, then the rest of the templates

File: weave-js/src/components/PagePanelComponents/Home/HomePreviewSidebar.tsx [hunk 4]
```diff
@@ -53,6 +58,53 @@ const DashboardTemplateItem = styled(LayoutElements.HStack)`
...
+HomeExpressionPreviewPartsWrapper.displayName =
+  'S.HomeExpressionPreviewPartsWrapper';
+
+type Template = {
+  display_name: string;
+  description: string;
+  op_name: string;
+};
+
+const SEED_BOARD_OP_NAME = 'py_board-seed_board';
```
Comment (tssweeney) [inline weave-js/src/components/PagePanelComponents/Home/HomePreviewSidebar.tsx:100]: This is fine for now, but it does feel like we could make this a bit more generic:
```
RECOMMENDED_BOARD_ORDER = ['py_board-open_ai_completions_monitor']
FALLBACK_BOARD_ID = 'py_board-seed_board'
```

Then the logic is essentially: recommend the first matching board in recommended_board_order, or the first non-fallback, else fallback. ... but this is fine - just calling out a way to generalize

**Review by tssweeney — APPROVED** [review]

**Pull request merged**